python/pyarrow/orc.py

```diff
@@ -234,8 +238,11 @@ class ORCWriter:
 """.format(_orc_writer_args_docs)
 
     is_open = False
+    file_handle = None
 
-    def __init__(self, where, *, file_version='0.12',
+    def __init__(self, where, *,
+                 filesystem=None,
```

Comment:
Instead of adding a `filesystem` argument, please add something like a `close_file=False` argument that will select whether to close the given file on `__exit__`.

Reply:
Really thanks! Fixed!